cmd/podman/trust.go

```diff
@@ -127,30 +128,26 @@ func showTrustCmd(c *cli.Context) error {
 		return nil
 	}
 
-	var policyContentStruct trust.PolicyContent
-	if err := json.Unmarshal(policyContent, &policyContentStruct); err != nil {
-		return errors.Errorf("could not read trust policies")
-	}
-	policyJSON, showOutputMap, err := trust.GetPolicy(policyContentStruct, systemRegistriesDirPath)
+	policyContentStruct, err := trust.GetPolicy(policyPath)
 	if err != nil {
-		return errors.Wrapf(err, "error reading registry config file")
+		return errors.Wrapf(err, "could not read trust policies")
 	}
+
 	if c.Bool("json") {
-		var outjson interface{}
+		policyJSON, err := trust.GetPolicyJSON(policyContentStruct, systemRegistriesDirPath)
```

Comment:
(Just a quick initial impression, not a full review:)

Splitting the formatting code into separate JSON/non-JSON variants is good, but why was it moved out of the CLI subpackage back into the backend subpackage?

Reply:
oh...misunderstood the meaning of the comment :fearful:

Reply:
Overall, the suggestion is that the _smarts_ (combining data from multiple sources into an useful result) should be in the backend, and the _UI_ (both parsing options, and formatting the computed data into output formats) should be in `cli/podman`.